ADMIN.adoc

```diff
@@ -0,0 +1,36 @@
+= Repository administrator responsibilities
+
+== Administrator rotation schedule
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be an administrator is voluntary.
+To volunteer to be an administrator, contact @cbudz or @bergerhoffer.
```

Comment:
Would you mind making these links to https://github.com/cbudz and https://github.com/bergerhoffer?